models/claims_preprocessing/acute_inpatient/final/acute_inpatient__summary.sql

```diff
@@ -77,14 +77,11 @@ group by 1
 
 , facility as (
     select
-        a.encounter_id
+          a.encounter_id
         , max(a.facility_id) as facility_id
-        , b.provider_organization_name
         , count(distinct facility_id) as npi_count
-    from {{ ref('acute_inpatient__institutional_encounter_id') }} a
-    left join {{ ref('terminology__provider') }} b
-    on a.facility_id = b.npi
-    group by 1,3
+    from {{ ref('emergency_department__int_institutional_encounter_id') }} a
```

Comment:
Was this change intentional? It seems weird to ref the ED institutional_encounter_id model instead of the AIP model.

Reply:
no it was not. Good catch!